### PR TITLE
Blakes greeting

### PR DESCRIPTION
**Pull request by brwjxxvi**

### CODE REVIEW

**Review by NicMcPhee — DISMISSED** [review]
Very nice – thanks!

**Review by Jeklund99 — DISMISSED** [review]
Great

**Comment by abbas070** [comment]
Good work!

**Review by NicMcPhee — DISMISSED** [review]
That looks fine – thanks.

**Review by Jeklund99 — DISMISSED** [review]
Looks good.

**Review by NicMcPhee — APPROVED** [review]
Looks great!

**Review by Jeklund99 — DISMISSED** [review]

**Review by NicMcPhee — APPROVED** [review]

**Review by DanteMillerDS — DISMISSED** [review]

**Comment by DanteMillerDS** [comment]
good job!

**Review by NicMcPhee — DISMISSED** [review]

File: src/main/java/hellos/Hellos.java [hunk 1]
```diff
@@ -34,9 +34,8 @@ public String generateOutput() {
     builder.append(natashaSaysHello());
     builder.append(nicSaysHello());
     builder.append(richardSaysHello());
-    builder.append(thomasSaysHello());
-
-
+    builder.append(thomasSaysHello())
```
Comment (helloworld12321) [inline src/main/java/hellos/Hellos.java]:
```suggestion
    builder.append(thomasSaysHello());
```

Oops, looks like you've got a missing semicolon 🙃

**Review by helloworld12321 — CHANGES_REQUESTED** [review]

**Review by NicMcPhee — DISMISSED** [review]

**Review by dahlgrenthomas — DISMISSED** [review]

**Review by ellst029 — DISMISSED** [review]

**Review by halleypaulson — DISMISSED** [review]

**Review by helloworld12321 — DISMISSED** [review]
Sorry it took me so long to re-review. Looks great, Blake! 🙂

**Review by lam00071 — DISMISSED** [review]
let's get this doneeeeeee

**Review by Jeklund99 — DISMISSED** [review]

**Review by JacobJenness — APPROVED** [review]

**Review by helloworld12321 — APPROVED** [review]
Here's hoping it'll go through!

**Pull request merged**